src/components/EstimatedLifecycleCost/Review/index.tsx

```diff
@@ -47,29 +49,49 @@ const EstimatedLifecycleCostReview = ({
 
   // Can be float or NaN
   const developmentCosts: { [key: string]: number } = {
-    year1: parseFloat(data.year1.development.cost),
-    year2: parseFloat(data.year2.development.cost),
-    year3: parseFloat(data.year3.development.cost),
-    year4: parseFloat(data.year4.development.cost),
-    year5: parseFloat(data.year5.development.cost)
+    year1: data.year1.development.isPresent
+      ? parseFloat(data.year1.development.cost)
+      : NaN,
```

Comment:
This feels a little strange, but is necessary because we are persisting the cost entered EVEN if the phase gets unchecked. We are doing this just in case a user unchecked a box they didn't mean to.
```
{
  isPresent: false,
  cost: '5'
}
```

If `isPresent` is `false`, that means it's unchecked and we shouldn't add the value. If it's `true`, we add it. We use `NaN` because it `parseFloat('')` is `NaN`, which is what the cost would be if the checkbox is unselected.